src/ranged.cpp

```diff
@@ -1436,6 +1436,13 @@ double player::get_weapon_dispersion(item *weapon, bool random) const
         int adj_recoil = recoil + driving_recoil;
         dispersion += rng( int(adj_recoil / 4), adj_recoil );
     }
+    
+    if ( weapon->has_flag("MOUNTED_GUN") &&
+        !m.has_flag_ter_or_furn( "MOUNTABLE", u.pos3() ) &&
```

Comment:
Needs to be `g->m` instead of `m` (`m` exists in the `game` class, this function is part of the `player` class and does not known about `m`).
Needs to be just `pos()` instead of `u.pos3()` (`pos` is function of the `player` class).

Also, you need to get the vehicle pointer `veh` before you can used it, just like it's gathered in game.cpp:
```C++
int vpart = -1;
vehicle *veh = g->m.veh_at( pos(), vpart );
```